pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools>=77.0", "setuptools-scm>=8.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mlx.jira-traceability"
+dynamic = ["version"]
+authors = [
+    {name = "Jasper Craeghs", email = "jce@melexis.com"},
```

Comment:
Now we can add all the past major authors here as well (Stein, Bavo, ...)

Reply:
Ah yes, let's do that for mlx.warnings and mlx.traceability. I've been the only author of this mlx.jira-traceability.